### ISSUE TITLE
Prepare messaging for Slack and backlog items affected on 'Experimental' tag removal

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-design-system#3247]
## What

Prepare a message for Slack, Prepare a message for backlog items tagged as 'Experimental'.

## Why
We want to merge the pull request: [Remove experimental status tag and references to experimental status on website](https://github.com/alphagov/govuk-design-system/pull/3198#top).

However, our users are familiar with the tag. We want to let them know what we suggest they do instead, our plans for next steps to acknowledge components/patterns where we don't have as much user research on their implementation as we would like.

## Who needs to work on this
Charlotte, Steve

## Who needs to review this
Anyone

## Done when
- [x] Comms message drafted for Slack
- [x] Comms message drafted for Accordion backlog item
- [x] Comms message drafted for Cookie banner backlog item
- [x] Comms message drafted for Exit this page backlog item
- [x] Comms message drafted for Notification banner backlog item
- [x] Comms message drafted for Tabs backlog item
- [x] Comms message drafted for Contact a department or service team backlog item
- [x] Comms message drafted for Exit a page quickly backlog item
- [x] Comms message drafted for Bank details backlog item
- [x] Comms message drafted for Page not found pages backlog item
- [x] Comms message drafted for There is a problem with the service pages (formerly technical problem pages) backlog item
- [x] Comms message drafted for Service unavailable pages backlog item
- [x] Comms message drafted for Cookies page backlog item
- [x] Comms message drafted for Telephone numbers backlog item
- [ ] Comms message drafted for Resources and tools 
- [x] Comms message drafted for Images

**Comment by CharlotteDowns** [comment]
## DRAFT:

Today we [removed the 'Experimental' tag](https://github.com/alphagov/govuk-design-system/issues/3194) from components, patterns, and guidance in the Design System 😌. 

The tag was being used to raise awareness that more research is needed to validate these components, patterns, and guidance. However, we recently published [new guidance on how to share findings from users](https://design-system.service.gov.uk/community/share-research-findings/) which we hope will make it easier to collect more information about how our Design System is being used across services.

These are the pages where you will no longer see an 'Experimental' tag but where we would still like more user research:
### Components

- [Accordion](https://design-system.service.gov.uk/components/accordion/)
- [Cookie banner](https://design-system.service.gov.uk/components/cookie-banner/)
- [Exit this page](https://design-system.service.gov.uk/components/exit-this-page/)
- [Notification banner](https://design-system.service.gov.uk/components/notification-banner/)
- [Tabs](https://design-system.service.gov.uk/components/tabs/)

### Patterns

- [Contact a department or service team](https://design-system.service.gov.uk/patterns/contact-a-department-or-service-team/)
- [Exit a page quickly](https://design-system.service.gov.uk/patterns/exit-a-page-quickly/)
- [Bank details](https://design-system.service.gov.uk/patterns/bank-details/)
- [Page not found pages](https://design-system.service.gov.uk/patterns/page-not-found-pages/)
- [There is a problem with the service pages](https://design-system.service.gov.uk/patterns/problem-with-the-service-pages/) (formerly technical problem pages)
- [Service unavailable pages](https://design-system.service.gov.uk/patterns/service-unavailable-pages/)
- [Cookies page](https://design-system.service.gov.uk/patterns/cookies-page/)
- [Telephone numbers](https://design-system.service.gov.uk/patterns/telephone-numbers/)

### Community

- [Resources and tools](https://design-system.service.gov.uk/community/resources-and-tools/)

### Guidance

- [Images](https://design-system.service.gov.uk/styles/images/)

**Comment by CharlotteDowns** [comment]
### Example Component message (assignee: @CharlotteDowns)

We have removed the 'Experimental' tag from components, patterns, and guidance in the Design System 😌. https://github.com/alphagov/govuk-design-system/issues/3194.

The tag was being used on the [x component] to raise awareness that more research is needed to validate it. However, we recently published [new guidance on how to share findings from users](https://design-system.service.gov.uk/community/share-research-findings/) which we hope will make it easier to collect more information about how our Design System is being used across services. 

If your team has used this component please let us know 💪🏻.

### Example Pattern message (assignee: @stevenjmesser)

We have removed the 'Experimental' tag from components, patterns, and guidance in the Design System 😌. https://github.com/alphagov/govuk-design-system/issues/3194.

The tag was being used on the [x pattern] to raise awareness that more research is needed to validate it. However, we recently published [new guidance on how to share findings from users](https://design-system.service.gov.uk/community/share-research-findings/) which we hope will make it easier to collect more information about how our Design System is being used across services. 

If your team has used this pattern please let us know 💪🏻.

### Example Guidance message (assignee: @CharlotteDowns)

We have removed the 'Experimental' tag from components, patterns, and guidance in the Design System 😌. https://github.com/alphagov/govuk-design-system/issues/3194.

The tag was being used on the images guidance to collect feedback about any of these areas:
- how you use images to give information to users
- examples of the types of images used in services
- how you create and choose images to use in your service
- how you implement images (for frontend performance)

We still want to get this feedback, we recently published [new guidance on how to share findings from users](https://design-system.service.gov.uk/community/share-research-findings/) which we hope will make it easier to collect information about how our Design System is being used across services.

**Comment by stevenjmesser** [comment]
Haven't been able to add a message for resources and tools because we collect that stuff via email. 

Should we point to the research and findings guidance from the contact partial?
